test/extended/etcd/etcd_storage_path.go

```diff
@@ -678,7 +679,10 @@ func JSONToUnstructured(stub, namespace string, mapping *meta.RESTMapping, dynam
 	return dynamicClient.Resource(mapping.Resource).Namespace(namespace), &unstructured.Unstructured{Object: typeMetaAdder}, nil
 }
 
-var protoEncodingPrefix = []byte{0x6b, 0x38, 0x73, 0x00}
+var (
+	protoEncodingPrefix = []byte{0x6b, 0x38, 0x73, 0x00}
+	cborPrefix          = []byte{0xd9, 0xd9, 0xf7, 0xa4}
```

Comment:
```suggestion
	cborPrefix          = []byte{0xd9, 0xd9, 0xf7}
```

The prefix is three bytes: `0xd9d9f7` (it's the encoding of the "self-described CBOR" tag, https://www.rfc-editor.org/rfc/rfc8949.html#section-3.4.6).

Reply:
Thanks. I have no idea where I got that extra byte from. Must have been some copy-paste mistake.

Reply:
`0xa4` would be the first byte of a length-4 map. Likely you were looking at an example of a real encoded object and picked it up there.